gcc/rust/lex/rust-lex.h

```diff
@@ -16,7 +16,18 @@ struct RAIIFile
   FILE *file;
 
 public:
-  RAIIFile (const char *filename) : file (fopen (filename, "r")) {}
+  RAIIFile (const char *filename)
+  {
+    if (strncmp (filename, "-", 1) == 0)
```

Comment:
I don't think the GNU coding style forces you to have all these braces for single statements if/else. You could have this a bit more compact, but that's cosmetic :)

Reply:
Oh right, I'm getting too used to writing Rust nowadays! I'll remove them, good catch